lib/util.js

```diff
@@ -88,16 +88,20 @@ Util.readFile = function (p, encoding) {
   return fs.readFileSync(p, encoding)
 }
 
-Util.readFileAsync = function (p, encoding) {
+Util.readFileAsync = async function (p, encoding) {
   if (encoding === undefined) encoding = 'utf8'
   Util.ensureDir(path.dirname(p))
   return new Promise((res, rej) => {
     fs.readFile(p, encoding, (err, data) => res(data))
   })
 }
 
-Util.writeFile = function (p, data) {
+Util.writeFile = async function (p, data) {
   Util.ensureDir(path.dirname(p))
```

Comment:
Presumably we'd want to ensure the directory even if the data is undefined, but this seems unclear to me.